src/components/Tutorial/MobileCodePreview.vue

```diff
@@ -132,7 +132,9 @@ $-preview-padding: 60px;
   /deep/ img:not(.file-icon) {
     border-radius: $border-radius;
     box-shadow: 0 0 3px rgba(0, 0, 0, 0.4);
-    min-height: 320px;
+    // Min-height matches the lowest height to support
+    // WatchOS' screen ratio on the narrowest client: 320px width
+    min-height: 243px;
```

Comment:
I don't totally understand where the `243px` value is coming from or if it's just a small enough value to work for a particular asset.

Either way...do we actually need to set a `min-height` value here at all? I know you didn't add it, but does it maybe make sense to just remove it entirely? (Not sure myself, just thinking out loud)

Reply:
Yeah, 243 is very specific 🤣 Removing the min-height doesnt seem to break anything from a first glance 🤔